.github/ISSUE_TEMPLATE/move-request--from-tf-contrib-to-addons-.md

```diff
@@ -0,0 +1,10 @@
+---
+name: Move request (from tf.contrib to addons)
+about: Describe this issue template's purpose here.
+title: ''
+labels: ''
+assignees: ''
+
+---
+
```

Comment:
We need know which function and why. Perhaps feature request template is just what we want here?

Reply:
How about we include a yes/no question in feature-request.md as follows:
Is this request for moving from` tf.contrib` to` addons`?

Reply:
@facaiy it was mentioned by @seanpmorgan in the issue #44 therefore created this but lets remove this then and add the question in feature-request.md

Reply:
I agree that they can be combined. Later on we can always remove that question from the template

Reply:
> How about we include a yes/no question in feature-request.md as follows:
> Is this request for moving from` tf.contrib` to` addons`?

Sounds great, so let's remove the file?

Reply:
done